src/storage/tracked.rs

```diff
@@ -0,0 +1,378 @@
+use std::iter::Enumerate;
+use std::marker::PhantomData;
+use std::ops::{Deref, DerefMut};
+use std::slice;
+
+use hibitset::{BitSet, BitSetLike};
+
+use {Component, DenseVecStorage, Index, MaskedStorage, Storage, UnprotectedStorage};
+
+#[derive(Copy, Clone, Debug, PartialEq, PartialOrd)]
+pub enum Change {
+    #[doc(hidden)]
+    None,
```

Comment:
Why is the `None` hidden? Also did you think just wrapping `Change` to `Option` or does that make it too verbose?

Reply:
Didn't document but basically that's because `None`s are filtered and just the internal layout. Using `Option` would need more space.